tools/shared/src/main/scala/org/scalajs/core/tools/io/VirtualFiles.scala

```diff
@@ -45,7 +45,10 @@ trait VirtualFile {
 object VirtualFile {
   /** Splits at the last slash and returns remainder */
   def nameFromPath(path: String): String = {
-    val pos = path.lastIndexOf('/')
+    val osName = System.getProperty("os.name")
```

Comment:
You cannot use `System.property("os.name")` here, because the Tools API is cross-compiled, and that will not work in the JS version. You can see that "bootstrap" tests are failing on the CI because of this. Besides, even on Windows, the `path` would be using forward slashes in many situations, so your fix could break other use cases.

What we typically do instead is to handle *both* `\` and `/`, on all platforms. One way to do it here would be
```scala
val pos = Math.max(path.lastIndexOf('/'), path.lastIndexOf('\\'))
```
or, to avoid the second call when it's not necessary:
```scala
val pos0 = path.lastIndexOf('/')
val pos =
  if (pos0 >= 0) pos0
  else path.lastIndexOf('\\')
```